edge/pkg/metamanager/metaserver/kubernetes/storage/sqlite/store.go

```diff
@@ -99,6 +100,22 @@ func (s *store) List(ctx context.Context, key string, opts storage.ListOptions,
 		if err != nil {
 			return err
 		}
+
+		labelSet := labels.Set(unstrObj.GetLabels())
+		if !opts.Predicate.Label.Matches(labelSet) {
+			continue
+		}
+
+		// TODO: only support metadata.name & metadata.namespace
+		fieldMap := make(map[string]string)
+		fieldMap["metadata.name"] = unstrObj.GetName()
+		if unstrObj.GetNamespace() != "" {
```

Comment:
Why should we judge that it is not empty, I understand that it should not match if it is empty.

Reply:
Thanks for  your review.  the judgement has been removed.

Reply:
In K8s, it also has the code to determine whether has the namespace. ref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/registry/generic/matcher.go#L25

I'm also not sure why need the `hasNamespaceField` here..

Reply:
the judgement is really useless. If  unsupported field-selector is specified, it will return empty whether the judgment is added or not.
In k8s, field selector not supported error occurs. so here i return  the same error.

Reply:
@fisherxu What do you think about it?